### DISCUSSION TITLE
Get Distinct Records via Directus API

### DISCUSSION THREAD

**Discussion by ILL35T** [directus/directus#6904] [Feature Requests]
The ability to select distinct records from a collection

**Comment by rijkvanzanten** [comment]
How do you think this should work? Allow a distinct function on the `fields` parameter (like `fields=distinct(rating)`), or a separate query param (`distinct=rating`), or something else?

**Reply by ILL35T** [reply]
I think we should follow our current pattern of using the fields as normal but by wrapping the field name with the word distinct as follows:
items/collection?fields=distinct(name),description,active&limit=10

I am currently pulling down 10000 records and finding the distinct name every time a new document is uploaded to my collection.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
